src/pages/iframes/comments/[dtpId].tsx

```diff
@@ -32,6 +44,23 @@ const CommentsIframePage: NextPage<CommentsIframePageProps> = ({
   );
 };
 
+const comments: Comment[] = [
+  {
+    id: 1,
+    text: "информация о верификации данных, если координаты изменены при обработке (координаты отличаются от заявленных ГИБДД, но прошли подтверждение модератором).",
+    user: "Павел Кучерягин",
+    date: new Date().toUTCString(),
+    avatarUrl:
+      "https://robohash.org/6ae852fa3a8b1c79dba3f7dc883c1760?set=set4&bgset=&size=200x200",
+  },
+  {
+    id: 2,
+    text: "Оставленная пользователями дополнительная/уточняющая информация",
+    user: "Anna Kravtz",
+    date: new Date().toUTCString(),
+  },
+];
+
 export const getServerSideProps: GetServerSideProps<
   CommentsIframePageProps
 > = async ({ params }) => {
```

Comment:
Now that we have `kebab-case`, we can probably replace 

```ts
const rawDtpId = `${params?.dtpId}`;
```

with

```ts
const rawDtpId = `${params?.["dtp-id"]}`;
```

Page path will become consistent with the rest of files without any external changes:

```
src/pages/iframes/comments/[dtpId].tsx
src/pages/iframes/comments/[dtp-id].tsx
```

Reply:
This is out of scope for this PR, so don’t worry about merging it 🙂